angr/analyses/calling_convention.py

```diff
@@ -295,6 +295,12 @@ def _generate_callsite_subgraph(self, func: 'Function', callsite_block_addr: int
         for _, dst, data in func.graph.out_edges(the_block, data=True):
             subgraph.add_edge(the_block, dst, **data)
 
+            # If the dst bb only contains a jump instruction, add the jump target as well.
+            dst_insns = func.get_block(dst.addr).capstone.insns
+            if len(dst_insns) == 1 and dst_insns[0].mnemonic == 'jmp':
```

Comment:
Can you please rewrite the check so that it does not explicitly use capstone or check against instruction mnemonics? I think the logic should be "if the target block contains only direct jump statements and has only one successor, include its successor." You can probably do this by check if the block.vex.statements is empty after lifting this block with opt_level=1.